### PR TITLE
[IGNORE] stories can directly import plugin.json files

### PR DESCRIPTION
**Pull request by juliepagano**
This PR helps address https://github.com/perses/perses/pull/1121#discussion_r1181679790. 

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

## UI Changes

- [x] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.
- [x] Code follows the [UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md).
- [x] Visual tests are stable and unlikely to be flaky. See [Storybook](https://github.com/perses/perses/tree/main/ui/storybook#visual-tests) and [e2e](https://github.com/perses/perses/tree/main/ui/e2e#visual-tests) docs for more details. Common issues include:
  - Is the data inconsistent? You need to mock API requests.
  - Does the time change? You need to use consistent time values or mock time utilities.
  - Does it have loading states? You need to wait for loading to complete.

### CODE REVIEW

File: ui/plugin-system/src/stories/shared-utils/decorators/WithPluginRegistry.tsx [hunk 1]
```diff
@@ -19,12 +19,8 @@ import {
...
 } from '@perses-dev/plugin-system';
 
-// NOTE: the aliases we use for components break these top level imports, so we
-// import relatively.
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const prometheusResource = require('../../../../../prometheus-plugin/plugin.json');
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const panelsResource = require('../../../../../panels-plugin/plugin.json');
+import prometheusResource from '@perses-dev/prometheus-plugin/plugin.json';
+import panelsResource from '@perses-dev/panels-plugin/plugin.json';
```
Comment (juliepagano) [inline ui/plugin-system/src/stories/shared-utils/decorators/WithPluginRegistry.tsx:23]: I initially said I'd do this in a separate PR because I thought I'd need to set up more aliases and refactor `storybook/config/main.js` a bunch, but it turns out the tweaks I made to the alias setup in https://github.com/perses/perses/pull/1121 fixed this for me. I just needed to switch the imports back.

**Review by sjcobb — APPROVED** [review]
Nice!

**Review by cndonovan — APPROVED** [review]
